MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md

```diff
@@ -6,11 +6,12 @@ MIP14c2-SP#: 4
 Author(s): @pyramidmaker
 Contributors: 
 Tags: Starknet
-Status: Formal Submission
+Status: Obsolete
 Date Proposed: 2023-01-12
-Date ratified: 
+Date ratified: N/A
 Forum URL: https://forum.makerdao.com/t/mip14c2-sp4-starknet-cu-budget-extension/19403/1
-Ratification Poll URL: TBD
+Ratification Poll URL: https://vote.makerdao.com/polling/QmXUiHqH
+Extra: This proposal has been marked as obsolete as per the changes introduced in [MIP0](https://mips.makerdao.com/mips/details/MIP0) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```

Comment:
For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.

Reply:
IMO there should be a difference between MIPs and subproposals, so I think we should keep this as is

Reply:
I see. Makes sense. Thank you!